tests/scripts/github-action-helper.sh

```diff
@@ -497,7 +508,7 @@ function test_csi_rbd_workload {
   kubectl create -f storageclass.yaml
   kubectl create -f pvc.yaml
   kubectl create -f pod.yaml
-  timeout 45 sh -c 'until kubectl exec -t pod/csirbd-demo-pod -- dd if=/dev/random of=/var/lib/www/html/test bs=1M count=1; do echo "waiting for test pod to be ready" && sleep 1; done'
+  timeout 90 sh -c 'until kubectl exec -t pod/csirbd-demo-pod -- dd if=/dev/random of=/var/lib/www/html/test bs=1M count=1; do echo "waiting for test pod to be ready" && sleep 1; done'
```

Comment:
how does this change affect the existing tests?

Reply:
> how does this change affect the existing tests?

The timeout is increased since its failing just due to flakiness since extra nfs csi pods are being deployed now.